pkg/strategy/elliottwave/draw.go

```diff
@@ -13,47 +13,49 @@ import (
 
 func (s *Strategy) InitDrawCommands(store *bbgo.SerialMarketDataStore, profit, cumProfit types.Series) {
 	bbgo.RegisterCommand("/draw", "Draw Indicators", func(reply interact.Reply) {
-		canvas := s.DrawIndicators(store)
-		if canvas == nil {
-			reply.Message("cannot render indicators")
-			return
-		}
-		var buffer bytes.Buffer
-		if err := canvas.Render(chart.PNG, &buffer); err != nil {
-			log.WithError(err).Errorf("cannot render indicators in ewo")
-			reply.Message(fmt.Sprintf("[error] cannot render indicators in ewo: %v", err))
-			return
-		}
-		bbgo.SendPhoto(&buffer)
+		go func() {
+			canvas := s.DrawIndicators(store)
+			if canvas == nil {
+				reply.Message("cannot render indicators")
```

Comment:
Since you're in the go routine, you need to use reply.Send (which sends the message directly)